src/main.ts

```diff
@@ -60,7 +60,7 @@ export async function createProject(options: Options) {
         task: () => createFirstGitCommit(targetDirectory, options),
       },
     ],
-    { rendererOptions: { collapseSkips: false, suffixSkips: true } },
+    { rendererOptions: { collapseSkips: false, suffixSkips: true, clearOutput: false } },
```

Comment:
What should it do? For me looking the same with or without it

Reply:
I have added it to fix output disappearing and forgot to remove it!
removing it now...